### PR TITLE
Fix html displaying and improved design

### LINKED ISSUE #2136: Organizer section showing html code and not following site design
Please fix the organizer section following design of site, e.g. add space at section end. Also fix html display.

screenshot from 2019-02-18 01-44-08

### PR DESCRIPTION
**Pull request by pranavkarthik10**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Does not display html in the section as it did previously. Also now there is extra space after content which matches the design from the rest of the page.

#### Changes proposed in this pull request:

- Changes within index.html in the orgdescription section
-
-

Fixes #2136

### CODE REVIEW

**Comment by pranavkarthik10** [comment]
@mariobehling Please review and merge!

**Comment by pranavkarthik10** [comment]
@bhaveshAn @aayusharora @geekyd @ParthS007 Please review

**Comment by pranavkarthik10** [comment]
@mariobehling Please review and merge!

**Comment by mariobehling** [comment]
Also, please help to review PRs and engage other devs to review your PRs to speed up things.

**Comment by pranavkarthik10** [comment]
Sure, I will definitely help others by reviewing their work as well! Please view the deployment here: https://limitless-gorge-66547.herokuapp.com.

**Comment by pranavkarthik10** [comment]
@mariobehling Please view the test page with changes at  https://limitless-gorge-66547.herokuapp.com/live/preview/pranav.karthik@icloud.com/FOSSASIASummit2016/ and the generator is deployed at https://limitless-gorge-66547.herokuapp.com.  
Screen Shot 2020-01-06 at 4 05 16 PM

File: src/backend/templates/event.hbs [hunk 1]
```diff
@@ -173,10 +173,11 @@
         <h1 class="section-header">Organized by {{eventurls.orgname}}</h1><br>
       </div>
       <section class="container">
+        <div class="row">
         <div class="col-sm-12 col-md-12 col-xs-12">
-          <p>
             {{eventurls.orgdescription}}
-          </p>
+            <br><br>
```
Comment (mariobehling) [inline src/backend/templates/event.hbs]: Instead of adding br tags the design should be adjusted and implemented in the CSS.

**Comment by pranavkarthik10** [comment]
@mariobehling Updated via css using a new class. Now there is extra padding to the bottom of the section.

**Comment by mariobehling** [comment]
Thanks. Please help to get Travis running in follow-up PRs.

**Comment by pranavkarthik10** [comment]
Thank you very much, I will definitely look into the Travis issue!

**Comment by pranavkarthik10** [comment]
@mariobehling  I have solved the travis issue as well in this PR: https://github.com/fossasia/open-event-wsgen/pull/2165

**Pull request merged**